pkg/compute/init.go

```diff
@@ -491,6 +505,45 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	return nil
 }
 
+// verifyDirectory indicates if the user wants to continue with the execution
+// flow when presented with a prompt that suggests the current directory isn't
+// empty.
+func verifyDirectory(out io.Writer, in io.Reader) (bool, error) {
+	files, err := os.ReadDir(".")
+	if err != nil {
+		return false, err
+	}
+
+	if len(files) > 0 {
+		dir, err := os.Getwd()
+		if err != nil {
+			return false, err
+		}
+
+		label := fmt.Sprintf("The current directory isn't empty. Are you sure you want to initialize a Compute@Edge project in %s? [y/n] ", dir)
+		cont, err := text.Input(out, label, in)
+		if err != nil {
+			return false, fmt.Errorf("error reading input %w", err)
+		}
+
+		contl := strings.ToLower(cont)
+
+		if contl == "n" || contl == "no" {
+			return false, nil
+		}
+
+		if contl == "y" || contl == "yes" {
+			return true, nil
+		}
+
+		// NOTE: be defensive and default to short-circuiting the execution flow if
+		// the input is unrecognised.
+		return false, nil
```

Comment:
@phamann maybe we should not default to short-circuiting because in order to support that flow we have to introduce a `--force` flag so people can automate skipping the verification step and that is a breaking change for the `compute init` command 😬

Reply:
Discussed with Andrew and he feels this isn't a breaking change.

Reply:
I am very relaxed about this because if it does trip up any customers, they have multiple remediation options and arguably were just flagging an error state.